src/gridtools/clang/verify.hpp

```diff
@@ -133,6 +133,28 @@ class verifier {
 
     return verified;
   }
+  template <class StorageType>
+  void printStorage(StorageType storage) {
+    using namespace gridtools;
```

Comment:
This is only working for 3d storages. I guess it is find for the verifier, no need to build a totally general printing, 
but can you add a simple static assert on the ndims == 3 of the storage info of the storage?

Reply:
we leave it as is for now and then generalize it in the near future